progs/triple_refinement.py

```diff
@@ -1,4 +1,4 @@
-from .polynomial import get_omega
+from honeybadgermpc.polynomial import get_omega
```

Comment:
There should be a test for this file. You will probably need to modify the test too. Run `pytest -v` from the container to see that all tests are passing. Anyway, the CI runs tests and it will fail if even a single test fails.

Reply:
should the test be moved over to the `progs/` folder, or should I leave it in test?

Reply:
you can create a `prog` folder in `tests` but then you will have to add a `conftest.py` as well in the `progs` folder which should import `from tests.fixtures import *`. You can basically follow the way things are in the `honeybadgermpc/benchmark` folder.